AirCasting/Settings/MyAccountViewSignOut.swift

```diff
@@ -41,18 +42,19 @@ private extension MyAccountViewSignOut {
     
     var signOutButton: some View {
         Button(action: {
-            guard networkChecker.connectionAvailable else {
-                alert = InAppAlerts.unableToLogOutAlert()
-                return
-            }
-            userState.currentState = .loggingOut
-            do {
-                try logoutController.logout {
-                    userState.currentState = .idle
+            measurementStreamStorage.accessStorage { storage in
```

Comment:
Wouldn't it make sense to check for the connection before checking for locationless sessions? Because if the connection is unavailable we can't move forward with logging out anyway.